README.md

```diff
@@ -2,46 +2,114 @@
+repositories.
 
 ## Building learning locker image
 
-To build a learning locker image, you have to pick the right [tag](https://github.com/LearningLocker/learninglocker/tags) to use and then build it using the following docker command (here with the `v2.6.2` tag):
+To build a learning locker image, you have to pick the right
+[tag](https://github.com/LearningLocker/learninglocker/tags) to use and then
+build it using the following docker command (here with the `v2.6.2` tag):
 
 ```bash
 $ docker build -t fundocker/learninglocker:v2.6.2 --build-arg LL_VERSION="v2.6.2" learninglocker/
 ```
 
 ## Building xapi-service image
 
-To build the xapi-service image, you have to pick the right [tag](https://github.com/LearningLocker/xapi-service/tags) to use and then build it using the following docker command (here with the `v2.2.15` tag):
+To build the xapi-service image, you have to pick the right
+[tag](https://github.com/LearningLocker/xapi-service/tags) to use and then build
+it using the following docker command (here with the `v2.2.15` tag):
 
 ```bash
 $ docker build -t fundocker/xapi-service:v2.2.15 --build-arg VERSION="v2.2.15" xapi/
 ```
 
 ## How to use the CI
 
-The CI configured in this repository will test if the image can be built successfully for a particular release, but it also pushes images to docker hub when this repository is tagged
+The CI configured in this repository will test if the image can be built
+successfully for a particular release, but it also pushes images to docker hub
+when this repository is tagged
 
 Here are the steps to follow to publish a new image:
 
-- Edit the `.circleci/releases.sh` file and update the tag version corresponding to the image you want to publish.
-- Commit your changes, submit a pull request and once merged into master you will be able to tag a new version.
+- Edit the `.circleci/releases.sh` file and update the tag version corresponding
+  to the image you want to publish.
+- Commit your changes, submit a pull request and once merged into master you
+  will be able to tag a new version.
 
-Tags must stick to the following patterns for the image release you want to publish:
+Tags must stick to the following patterns for the image release you want to
+publish:
 
-- `learninglocker-[RELEASE_TAG]` if you want to publish a new learning locker image (_e.g._ `learninglocker-v2.6.2`),
-- `xapi-service-[RELEASE_TAG]` if you want to publish a new xapi-service image (_e.g._ `xapi-service-v2.2.15`).
+- `learninglocker-[RELEASE_TAG]` if you want to publish a new learning locker
+  image (_e.g._ `learninglocker-v2.6.2`),
+- `xapi-service-[RELEASE_TAG]` if you want to publish a new xapi-service image
+  (_e.g._ `xapi-service-v2.2.15`).
+
+### Using the release script
+
+If you have more than one release to publish, we also provide the `bin/release`
+Bash script to automate the whole releasing process.
+
+#### Requirements
+
+The `bin/release` script requires to install the two following dependencies
```

Comment:
In the requirements you also need a github token, maybe you should explain how to obtain one and how to export it to use it here.

Reply:
This is not a requirement as you are allowed to make 60 requests per hour on GitHub's public API without being authenticated (this is more than sufficient to use this tool):

> For unauthenticated requests, the rate limit allows for up to 60 requests per hour. Unauthenticated requests are associated with the originating IP address, and not the user making requests.

Source: https://developer.github.com/v3/#rate-limiting

Reply:
Ok but maybe you should had something in the readme about this token. You see it in the option when you run the `usage` command and I don't know what I should do without your explanation.

Reply:
Done. WDYT?